docs/data-cookbooks/browse-generation.md

```diff
@@ -534,13 +539,13 @@ In the provided example, a payload being passed to ```MoveGranules``` should be
   ]
 ```
 
-This list is the list of granules ```MoveGranules``` will act upon to move from the staging directory to the configured buckets.
+This list is the list of granules ```FilesToGranules``` will act upon to move from the staging directory to the configured buckets.
```

Comment:
`FilesToGranules` does not perform a move.

Reply:
d3bf4d6